js/palette.js

```diff
@@ -498,107 +501,107 @@ function PaletteModel(palette, palettes, name) {
+            case "text":
+                label = _("text");
+                break;
+            case "drumname":
+                label = _("drum");
+                break;
+            case "effectsname":
+                label = _("effect");
+                break;
+            case "solfege":
+                label = i18nSolfege("sol");
+                break;
+            case "eastindiansolfege":
+                label = _("sargam");
+                break;
+            case "scaledegree2":
+                label = _("scale degree");
+                break;
+            case "modename":
+                label = _("mode name");
+                break;
+            case "invertmode":
+                label = _("invert mode");
+                break;
+            case "voicename":
+                label = _("voice name");
+                break;
+            case "customNote":
+                label = _("custom pitch");
+                break;
+            case "temperamentname":
+                //TRANS: https://en.wikipedia.org/wiki/Musical_temperament
+                label = _("temperament");
+                break;
+            case "accidentalname":
+                //TRANS: accidental refers to sharps, flats, etc.
+                label = _("accidental");
+                break;
+            case "notename":
+                label = "G";
+                break;
+            case "intervalname":
+                label = _("interval name");
+                break;
+            case "boolean":
+                label = _("true");
+                break;
+            case "number":
+                label = NUMBERBLOCKDEFAULT.toString();
+                break;
+            case "less":
+            case "greater":
+            case "equal":
+                // Label should be inside _() when defined.
                 label = protoBlock.staticLabels[0];
-                if (label === "") {
-                    if (blkname === "loadFile") {
-                        label = _("open file");
+                break;
+            case "namedarg":
+                label = "arg " + arg;
+                break;
+            case "outputtools":
+                label = _("pitch converter");
+                break;
+            default:
+                if (blkname != modname) {
+                    // Override label for do, storein, box, and namedarg
+                    if (blkname === "storein" && block.defaults[0] === _("box")) {
+                        label = _("store in");
+                    } else if (blkname === "storein2") {
+                        if (block.staticLabels[0] === _("store in box")) {
+                            label = _("store in box");
+                        } else {
+                            label = _("store in") + " " + block.staticLabels[0];
+                        }
                     } else {
-                        label = blkname;
+                        label = block.defaults[0];
```

Comment:
curious about this change.

Reply:
Its line 578 from the older file. I didn't change it. Maybe some difference in formatting is comparing this line with line 586.